waspc/src/Wasp/Generator/ServerGenerator/JsImport.hs

```diff
@@ -29,6 +30,14 @@ getJsImportStmtAndIdentifier ::
   (JsImportStatement, JsImportIdentifier)
 getJsImportStmtAndIdentifier pathFromImportLocationToExtCodeDir = JI.getJsImportStmtAndIdentifier . extImportToJsImport pathFromImportLocationToExtCodeDir
 
+getAliasedJsImportStmtAndIdentifier ::
```

Comment:
Have you looked at `extImportToImportJson` and maybe provide a fn like that for aliased imports?

Reply:
Do you mean in addition to this one? Can we just add that once we need it, or do you think it is needed now?